Fundamentals/Code/Creating-Forms/index.md

```diff
@@ -65,13 +65,13 @@ The view can be built with standard MVC helpers:
 ```
 
 ### Adding the controller
-Finally, we're going to add the controller. Create a new empty class in the `/Controllers` folder (if the folder doesn't already exist, create it at the root of the website). Name it `ContactFormController` and make it inherit from `SurfaceController`. Inheriting from `SurfaceController` requires that you call its base constructor, most IDE's can do this automatically for you.
+Finally, we're going to add the controller. Create a new empty class in the `/Controllers` folder (if the folder doesn't already exist, create it at the root of the website). Name it `ContactFormController` and make it inherit from `SurfaceController`. Inheriting from `SurfaceController` requires that you call its base constructor, most IDE's: Integrated Development Environments, can do this automatically for you.
```

Comment:
I think the issue might be that the acronyms has `'s` after it, so the formatting rule can't really figure it out 😅 

How about rephrasing the sentence to:
> Inheriting from `SurfaceController` requires that you call its base constructor. If you are using an IDE: Integrated Development Environments, this can be done automatically.

Reply:
@sofietoft I think so too 😅. And rephrasing the sentence is a good solution 😄 !